mmdet/models/seg_heads/panoptic_fpn_head.py

```diff
@@ -0,0 +1,129 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+from mmcv.runner import BaseModule, ModuleList
+
+from ..builder import HEADS
+from .base_semantic_head import BaseSemanticHead
+
+
+class ConvUpsample(BaseModule):
+    """The subnet of Panoptic FPN Head."""
+
+    def __init__(self,
```

Comment:
1. docstring.
2. is there any case that num_layers will differ from num_upsample?

Reply:
num_layers=i if i > 0 else 1,
num_upsample=i if i > 0 else 0,